models/app.model.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection");
+const format = require("pg-format");
+
+exports.topicsFunc = () => {
+    return db.query(`
+        SELECT * FROM topics
+    `)
+}
```

Comment:
Your model should be handling your query to the database and serving up what is necessary to pass back to the controller. Please make sure your model is returning the result of the query.